INSTALL.md

```diff
@@ -20,67 +23,55 @@ of this file.
 
 In order to compile LCDproc, you'll need the following programs:
 
-* A C compiler which supports C99, we recommend GCC. Most Linux or BSD systems
-  come with GCC.
+* A C compiler which supports C99, we recommend GCC. Most Linux or BSD systems come with GCC.
 
```

Comment:
I'd prefer if we could keep the line breaks as they are. Most variants of markdown support that.